### PR TITLE
Fixing Sourceforge download link

### LINKED ISSUE #8: Sourceforge mirror unreachable
Similar to #5, the https://cfhcable.dl.sourceforge.net/project/astyle/astyle/astyle%203.1/astyle_3.1_linux.tar.gz link seems to be unreachable.

As per this [documentation](https://sourceforge.net/p/forge/documentation/Downloading%20files%20via%20the%20command%20line/), it seems like you can use a slightly different link to have it auto-select a mirror-- this seems to work: https://sourceforge.net/projects/astyle/files/astyle/astyle%203.1/astyle_3.1_linux.tar.gz/download?use_mirror=cfhcable

Also "cfhcable" no longer appears on the [Mirrors](https://sourceforge.net/p/forge/documentation/Mirrors/) list...

### PR DESCRIPTION
**Pull request by danwahl**
Closes #8

### CODE REVIEW

File: artistic-style-check.sh [hunk 1]
```diff
@@ -48,7 +48,9 @@ while IFS='' read -r targetPath && [[ -n "$targetPath" ]]; do
     # Check if it's a file (find matches on pruned folders)
     if [[ -f "$filename" ]]; then
       echo "::group::Checking $filename"
-      if ! diff --strip-trailing-cr "$filename" <(astyle --options="$optionsFilePath" --dry-run <"$filename"); then
+      diffOutput=$(diff --strip-trailing-cr "$filename" <(astyle --options="$optionsFilePath" --dry-run <"$filename"))
+      if [[ -n "$diffOutput" ]]; then
+        echo "$diffOutput"
```
Comment (per1234) [inline artistic-style-check.sh:53]: @danwahl please limit the scope of the changes in the PR exclusively to the stated purpose of fixing the download URL. You can submit separate PRs for other proposals.

**Review by per1234 — CHANGES_REQUESTED** [review]

**Comment by danwahl** [comment]
Closed in favor of #10
